features/notifications/notifications-init.ts

```
@@ -131,99 +142,111 @@ async function handlingNonDecryptedExpoNewMessageNotification(args: {
+                  identifier: attachment.url,
+                  type: "image",
+                  url: attachment.url,
+                })),
+              }),
+          },
+          trigger: null,
+        })
+      },
     })
+  } catch (error) {
+    throw error
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)